schema-generator/src/main/resources/default_values_prov.sql

```diff
@@ -51,7 +51,7 @@ VALUES(8, 2, 6, 'Captive-Portal', decode('504b030414000808080087b2f6500000000000
+VALUES(10, 2, 9, 'TipWlan-rf', decode('504b0304140008080800e87e4e510000000000000000000000000100000061ed564d73da3010fd2f3a3b330e8953ea1b98d0301348066872cc087b819d9157ae24877c0cffbd2be3900f273da43db41d5f30969e76dfaedebef1a3c87506eac6dd172062315d269a96b82a8d74a84904c2d42b635988f851a08dbe9d3df87f6fce9d2ac8815cf338efceb1b856920ef825100b90a9a6113930b75289f8300c03b1d42685592a69501aa4d595c6d487cdd0ca8582cc0772367176be3660d75a65223e89a2a32810e95a1281ea4bca3698b9359f42db0dc74c321039e67acc3c79d16d74ff7ebed17e55de4dca3c51c87c6d4d202f95c3545a3795cec365e93cd43f925d86192848abaae2a5541678935f6f2bd633708e69db465f7a4d4820807645c5ce941cc6f2eed0c08f1228bd9f013727ab58ed7606752fc7a8145a5ff6d657407255f5fb355d73978052337c80c1a2c9853117662ca994ea4aaa12eafa9e78dc566bf1c1d7903314462f600ab6d06461dff5df8d9a563d1fa0e52289bbf9c702039a627e77a937603e1be8b0bbf5e2b4ae579c3e5d10954a05820057eb85ae94798e1eb0537923d3e405b077e9a16fc721479ae18abcee0fba5125c55ec1b7daa96e15c9737dba42869c30a4008339f0b40cf8a712e0e96bfdd41370a68b0f659834215c96460b43a5b5d95fc388067d4efb257a7777ce3c46f4acd06ee861748d4b7c11e1127894c9b13e451cbd83782f8a5e58cbe4ea519d5c4c6eae47c391d85fc887854d6586baff1ac38d5489ce8bd23df72833bae0a4645eb23b0aab9e9f6b990d7996354be738dc7252b49d4cbbe37fc0e83aadd1b546d71a5d6b74bf34ba4ed3e8a2dae8fcd7dcf7bfdde5dacfb9d6e55a976b5deed39f73dee5ce5b976b5dae75b9d6e5fe4b97dba97f890ae6bb246c5bdb9f504b070880dd756d7a020000dd130000504b01021400140008080800e87e4e5180dd756d7a020000dd13000001000000000000000000000000000000000061504b050600000000010001002f000000a90200000000', 'hex'),
```

Comment:
@mikehansen1970 Do you know what checks or tests might need to be run against this?

Reply:
This was the change that I needed Rahul's help with this morning. The hex string is generated and copied from the Postgres tables directly